skpro/datatypes/_convert.py

```diff
@@ -234,24 +254,24 @@ def convert_to(
     from_type = infer_mtype(obj=obj, as_scitype=as_scitype)
     as_scitype = mtype_to_scitype(from_type)
 
-    # if to_type is a list, we do the following:
-    # if on the list, then don't do a conversion (convert to from_type)
-    # if not on the list, we find and convert to first mtype that has same scitype
-    if isinstance(to_type, list):
-        # no conversion of from_type is in the list
-        if from_type in to_type:
-            to_type = from_type
-        # otherwise convert to first element of same scitype
-        else:
-            same_scitype_mtypes = [
-                mtype for mtype in to_type if mtype_to_scitype(mtype) == as_scitype
-            ]
-            if len(same_scitype_mtypes) == 0:
-                raise TypeError(
-                    "to_type contains no mtype compatible with the scitype of obj,"
-                    f"which is {as_scitype}"
-                )
-            to_type = same_scitype_mtypes[0]
+    # # if to_type is a list, we do the following:
```

Comment:
why is this commented out? Can this not be entirely removed?

Reply:
impacts both `skpro` and `sktime`

Reply:
the comment out code was not used at all in the sktime `_convert.py` file. will remove it. I only commented it out and left it there to see if it would break anything if it was commented